packages/mui-nested-menu/src/definitions.ts

```diff
@@ -1,8 +1,12 @@
+import { SxProps } from '@mui/system';
```

Comment:
same here

Reply:
Changed to `'@mui/system/styleFunctionSx'`